content/Circular_Intersection.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Circular Intersections
+draft: true
+tags:
+  - math
+---
+# Circular Intersections
+## How I started this problem
+One day I was writing my school HW where I had to fill out a venn diagram contrasting to people. Because the middle section soon ran out of space, I thought: *How can I make all three sections the same?*
+## Problem
+When taking two congruent circles, when are the area's of the intersection of the two circles equal to the un intersected section of one circle?
+
+![[venn_diagram2.png|400]]
+## Solution
+
```

Comment:
need to add something like: " now let's draw two identical circle P & Q with radii as 1 for simplicity. We will denote ... as k, ... as $\theta$ ... "